client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx

```diff
@@ -155,7 +146,7 @@ const DnsAddNew = React.createClass( {
 
 		return (
 			<form className="dns__add-new">
-				<div className={ classes }>
+				<div className="dns__form-content">
```

Comment:
Because the class name is not allowed, I get a warning. It was a mistake in the initial implementation.